logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationStatus.kt

```diff
@@ -0,0 +1,9 @@
+package com.wire.kalium.logic.data.conversation
+
+enum class MutedConversationStatus {
+    ALL_MUTED, ONLY_MENTIONS_ALLOWED, ALL_ALLOWED;
```

Comment:
Isn't it possible to use `MutedStatus` from network/`MemberUpdateRequest` in order to have only one source of truth for the enums? Or maybe have the enum class somewhere else where it could be used more than once? just a thought tho

Reply:
Thanks for the comment, why we are having this duplicated it's related to using this `MutedConversationStatus ` on Android Reloaded, we will use these values to change it from the UI. So in this way we are not exposing the `MutedStatus` from `network/MemberUpdateRequest` 😓